// socket-controllers/types/enums/ParameterType.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ParameterType = void 0;
var ParameterType;
(function (ParameterType) {
    ParameterType[ParameterType["CUSTOM"] = 0] = "CUSTOM";
    ParameterType[ParameterType["CONNECTED_SOCKET"] = 1] = "CONNECTED_SOCKET";
    ParameterType[ParameterType["MESSAGE_BODY"] = 2] = "MESSAGE_BODY";
    ParameterType[ParameterType["MESSAGE_ACK"] = 3] = "MESSAGE_ACK";
    ParameterType[ParameterType["SOCKET_QUERY_PARAM"] = 4] = "SOCKET_QUERY_PARAM";
    ParameterType[ParameterType["SOCKET_IO"] = 5] = "SOCKET_IO";
    ParameterType[ParameterType["SOCKET_ID"] = 6] = "SOCKET_ID";
    ParameterType[ParameterType["SOCKET_REQUEST"] = 7] = "SOCKET_REQUEST";
    ParameterType[ParameterType["SOCKET_ROOMS"] = 8] = "SOCKET_ROOMS";
    ParameterType[ParameterType["NAMESPACE_PARAMS"] = 9] = "NAMESPACE_PARAMS";
    ParameterType[ParameterType["NAMESPACE_PARAM"] = 10] = "NAMESPACE_PARAM";
})(ParameterType = exports.ParameterType || (exports.ParameterType = {}));
